### PR TITLE
added albums html file with boilerplate and ID

### PR DESCRIPTION
**Pull request by Cgatewood55**

### CODE REVIEW

File: albums.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel= "stylesheet" href= "/main.css">
+    <title>Document</title>
```
Comment (ghost) [inline albums.html]: Can you change the title to "Albums"? thx

Comment (ledugani) [reply albums.html]: fullsizerender-1

Comment (ghost) [reply albums.html]: 👍 +1

**Comment by ledugani** [comment]
👍

**Comment by AlexOrga** [comment]
👍

**Comment by ghost** [comment]
👍

**Comment by ghost** [comment]
👍

**Pull request merged**